test/AppKernel.php

```diff
@@ -11,12 +11,18 @@ class AppKernel extends Kernel
     private $configuration;
     private $enabledPlugins;
 
-    public function __construct(array $configuration, array $enabledPlugins)
+    private $cacheDir;
+    private $logsDir;
+
+    public function __construct(array $configuration, array $enabledPlugins, $uniqueIdentifier, $cacheDir, $logsDir)
     {
-        parent::__construct('test', true);
+        parent::__construct('test' . $uniqueIdentifier, true);
```

Comment:
This shouldn't be necessary anymore, since having a random cache dir already solves the isues.